app/lib/features/spaces/widgets/public_spaces_selector.dart

```diff
@@ -219,28 +276,19 @@ class PublicSpaceItem extends ConsumerWidget {
               child: ListTile(
                 onTap: () => onSelected(
                   space,
-                  withInfo.valueOrNull,
                 ),
-                leading: withInfo.when(
-                  data: (data) => data != null
-                      ? ActerAvatar(
-                          mode: DisplayMode.Space,
-                          avatarInfo: AvatarInfo(
-                            uniqueId: spaceId,
-                            displayName: data.spaceProfileData.displayName,
-                            avatar: data.spaceProfileData.getAvatarImage(),
-                          ),
-                        )
-                      : fallbackAvatar(),
+                leading: profileInfo.when(
+                  data: (profile) => ActerAvatar(
+                    mode: DisplayMode.Space,
+                    avatarInfo: AvatarInfo(
+                      uniqueId: space.roomIdStr(),
+                      displayName: profile.displayName,
+                      avatar: profile.getAvatarImage(),
+                    ),
+                  ),
                   error: (e, s) {
                     _log.severe('loading failed', e, s);
-                    return ActerAvatar(
-                      mode: DisplayMode.Space,
-                      avatarInfo: AvatarInfo(
-                        uniqueId: spaceId,
-                        displayName: spaceId,
-                      ),
-                    );
+                    return fallbackAvatar();
```

Comment:
What's the reason of using custom widget method for fallback when ```ActerAvatar``` should/will manage its fallback ? If there's a bug, we should open this issue on plugin repo...

Reply:
just a separation of code.